adventure/api.py

```diff
@@ -75,5 +78,16 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
-    return JsonResponse({'error': "Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    player_id = player.user.id
+    player_uuid = player.uuid
+    data = json.loads(request.body)
+    message = data['message']
+    room = player.room()
+    player_UUIDs = room.player_UUIDs(player_id)
+    for p_uuid in player_UUIDs:
+        pusher.trigger(f'p-channel-{p_uuid}', u'broadcast',
```

Comment:
Whoa. This is all it takes, huh? :))

Reply:
Lol, pretty much. Outside of the actual pusher config on your account dashboard.